config/config/wc-config.yaml

```diff
@@ -84,41 +131,85 @@ falco:
     tolerations: []
     affinity: {}
     nodeSelector: {}
+    ## if type=slack falco.alerts.slackWebhook must be set in the secrets yaml file
 
+## Prometheus configuration
+##
 prometheus:
   remoteWrite:
-    # Same as 'influxDB.users.wcWriter'.
+    ## Same as 'influxDB.users.wcWriter'.
     user: wcWriter
+
   storage:
     size: 5Gi
+
+  ## When to remove metrics from local storage.
+  ##
   retention:
     size: 4GiB
     age:  3d
+
+  ## Define resources requests and limits for single Pods.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources:
     requests:
       memory: 1Gi
       cpu: 300m
     limits:
       memory: 2Gi
-      cpu: "1"
+      cpu: "1" ## Must be a string
```

Comment:
The API-server complains. I'm not sure exactly why though, haven't looked into it, but I'm guessing that this is set on the Prometheus object and that the CRD requires it to be a string for some reason.

Reply:
Actually, it looks like string or integer both should be fine, but no floats though. Maybe that is just in newer versions or something?
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/crds/crd-prometheuses.yaml#L1987

Reply:
I just looked at it. And it seems like the CRD accepts both int and string in the new version (0.45) but I think we use 0.44 which only accepts a string. So that must be it.